src/Microsoft.AspNetCore.Server.Kestrel.Core/Internal/Http/Frame.cs

```diff
@@ -410,15 +419,15 @@ private void CancelRequestAbortedToken()
         /// <summary>
         /// Immediate kill the connection and poison the request and response streams.
         /// </summary>
-        public void Abort(Exception error)
+        public void Abort(Exception error, bool abortOutputWithError = false)
```

Comment:
It's not really about force-closing the output, it's just about propagating the error. With that in mind, do you have another suggestion?

I wouldn't want to make it a required parameter, since it's called with `true` in a single place. How about an overload?